services/provider/server/server.go

```diff
@@ -361,9 +361,9 @@ func (s *OCSProviderServer) GetDesiredClientState(ctx context.Context, req *pb.G
 			kubeResourceBytes := mustMarshal(kubeResource)
 			response.KubeResources = append(response.KubeResources, kubeResourceBytes)
 			response.KubeObjects = append(response.KubeObjects, &pb.KubeObject{
-				Bytes: kubeResourceBytes,
+				Bytes:           kubeResourceBytes,
 				ReconcilePolicy: pb.KubeObjectReconcilePolicy_CreateOrUpdate,
-				UpdateStrategy: pb.KubeObjectUpdateStrategy_UpdateWithReplaceAsFallback,
+				UpdateStrategy:  pb.KubeObjectUpdateStrategy_UpdateWithReplaceAsFallback,
```

Comment:
Please don't do that. Our build system enforces code linting and formatting. If it didn't raise an error then you should not be changing this in an unrelated PR